daemon/algod/api/server/v2/utils.go

```diff
@@ -49,6 +49,10 @@ func serviceUnavailable(ctx echo.Context, internal error, external string, log l
 	return returnError(ctx, http.StatusServiceUnavailable, internal, external, log)
 }
 
+func timeout(ctx echo.Context, internal error, external string, log logging.Logger) error {
```

Comment:
We should update the documentation for this endpoint mentioning that it can return a 408.

I don't recall the rational, but Indexer uses a 500 for time out errors.